sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,12 @@
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN
+('spell_four_horsemen_consumption',
+'spell_rajaxx_thundercrash',
+'spell_gen_arcane_charge',
+'spell_pet_dk_gargoyle_strike');
+
+INSERT INTO `spell_script_names` VALUES
```

Comment:
Codestyle: TC prefers that the column names are included in SQL queries:
```sql
INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
```

Reply:
Fixed. Thanks!